Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -2703,7 +2703,7 @@ def __init__(self, directory_entries, parent_directory_path=None):
     def _get_attachments_names(self):
         names = []
         for attachment in self.attachments:
-            names.append(attachment.DisplayName)
+            names.append(attachment.DisplayName or attachment.Filename)
```

Comment:
When will the attachment not have a DisplayName? 
Do we have a case like that in the test data?

Reply:
Yes (`html_attachment.msg` in `test_data` folder). And we're testing it